src/qml/components/BlockClock.qml

```diff
@@ -81,6 +81,15 @@ Item {
             root.paused = !root.paused
             nodeModel.pause = root.paused
         }
+        Rectangle {
+            visible: root.visualFocus
```

Comment:
```suggestion
            visible: root.activeFocus && (root.focusReason == Qt.TabFocusReason || root.focusReason == Qt.BacktabFocusReason)
```

Reply:
Item doesn't have the visualFocus property.  The suggested change is equivalent.

Reply:
Thanks! fixed, A follow-up needs to figure out how to enable keyboard click to signal clicked to the mouse area when tab focused

<img width="263" alt="Screen Shot 2023-02-16 at 1 27 07 PM" src="https://user-images.githubusercontent.com/23396902/219455745-f4a052ba-45d6-4c66-b2e9-61c5db0207fb.png">